packages/terra-disclosure-manager/src/terra-dev-site/doc/disclosure-manager/DisclosingContentWithAHeader.4.doc.mdx

```diff
@@ -44,3 +44,29 @@ const DisclosedComponent = () => {
 ```
 
 This will register the header information on mount.
+
+## Providing a Title on Disclose
+It has been observed that there can be a 'pop-in' affect with the header render in slow domains and using the `DisclosureManagerHeaderAdapter`. It is recommended to provide a `title` parameter to the disclose function that will render the disclosed content to ensure the header is rendered immediately. This will render the header with the title, and then the `DisclosureManagerHeaderAdapter` can be used to add additional button content into the header.
```

Comment:
Probably worth it to throw in a note here saying that the provided title will supersede any title value defined by the header adapter.

Reply:
Updated: https://github.com/cerner/terra-framework/pull/1014/commits/717d01e3e9e8b41248cadb72d34c6a5763948e31